### PR TITLE
Adding test for converters

### PR DESCRIPTION
**Pull request by Nekio21**

### CODE REVIEW

**Review by Nekio21 — COMMENTED** [review]
SUPER CODE. THE BEST

File: src/test/java/umk/mat/pajda/ProjektZespolowy/converters/UserTest.java [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+import org.mockito.Mock;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.context.ActiveProfiles;
+import umk.mat.pajda.ProjektZespolowy.DTO.*;
+import umk.mat.pajda.ProjektZespolowy.entity.User;
+import umk.mat.pajda.ProjektZespolowy.misc.UserConverter;
+import umk.mat.pajda.ProjektZespolowy.repository.UserRepository;
+
+@ActiveProfiles("tests")
+public class UserTest {
```
Comment (ThePapiok) [inline src/test/java/umk/mat/pajda/ProjektZespolowy/converters/UserTest.java:20]: should be UserConverterTest

File: src/test/java/umk/mat/pajda/ProjektZespolowy/converters/ReviewArchiveConverterTest.java
Comment (ThePapiok) [inline src/test/java/umk/mat/pajda/ProjektZespolowy/converters/ReviewArchiveConverterTest.java:1]: You can delete this class if we don't use reviewarchive

File: src/main/java/umk/mat/pajda/ProjektZespolowy/entity/ReviewArchive.java
Comment (ThePapiok) [inline src/main/java/umk/mat/pajda/ProjektZespolowy/entity/ReviewArchive.java:1]: Why is it here ?

**Review by ThePapiok — APPROVED** [review]

**Pull request merged**